course-definition.yml

```
@@ -483,5 +489,8 @@ stages:
 
       Your program must exit with 0 if the input matches the given pattern, and 1 if not.
     marketing_md: |
-      In this stage, you'll add support for nested backreferences. You'll implement support for multiple `\d` that references capturing groups in other capturing groups.
+      In this stage, you'll add support for nested backreferences. You'll implement support for multiple `\N` that references capturing groups in other capturing groups.
```

```suggestion
      In this stage, you'll add support for nested backreferences.
```

(The description is kinda confusing, and I can't come up with a better version - maybe the example is sufficient here?)